open-sphere-base/core/src/main/java/io/opensphere/core/control/keybinding/ControlTab.java

```diff
@@ -0,0 +1,194 @@
+        Color color = Color.WHITE;
+        label.setTextFill(color);
+        label.setOpacity(75);
+
+        final String fontName = "icons-" + text.getFont().getFontName().replaceAll("\\s", "-");
+        label.getStyleClass().addAll(fontName);
+        label.setStyle("-fx-font-size: 16 px;");
+
+        theButton.setGraphic(label);
+        theButton.setStyle("-fx-background-color:#0066cc;-fx-opacity: 1;");
+        theButton.setDisable(true);
+        theButton.setMaxSize(22, 22);
+        return theButton;
+    }
+
+    @SuppressWarnings("unused")
+    public GridPane createBlankPane(int width, int height, String header)
+    {
+        GridPane thePane = new GridPane();
+        thePane.setMinWidth(width);
+        thePane.setMinHeight(height);
+        thePane.setStyle(
+                "-fx-background-color : #363636;-fx-border-color: #004080;-fx-border-width: 4;-fx-background-radius: 8 8 8 8;"
+                        + "    -fx-border-radius: 8 8 8 8;");
+
+        myDisplayArea = new GridPane();
+        ColumnConstraints col1Constraints = new ColumnConstraints();
+        col1Constraints.setPercentWidth(47);
+        ColumnConstraints col2Constraints = new ColumnConstraints();
+        col2Constraints.setPercentWidth(6);
+        ColumnConstraints col3Constraints = new ColumnConstraints();
+        col3Constraints.setPercentWidth(47);
+        myDisplayArea.getColumnConstraints().addAll(col1Constraints, col2Constraints, col3Constraints);
+        myDisplayArea.setVgap(5);
+        myDisplayArea.setPadding(new Insets(0, 10, 10, 10));
+        thePane.add(createHeader(header), 0, 0);
+        thePane.add(myDisplayArea, 0, 1);
+
+        return thePane;
+    }
+
+    public GridPane createBlankPane(String header)
+    {
+        GridPane thePane = new GridPane();
+        thePane.setStyle(
+                "-fx-background-color : #363636;-fx-border-color: #004080;-fx-border-width: 4;-fx-background-radius: 8 8 8 8;"
+                        + "    -fx-border-radius: 8 8 8 8;");
+
+        myDisplayArea = new GridPane();
+        ColumnConstraints col1Constraints = new ColumnConstraints();
+        col1Constraints.setPercentWidth(47);
+        ColumnConstraints col2Constraints = new ColumnConstraints();
+        col2Constraints.setPercentWidth(6);
+        ColumnConstraints col3Constraints = new ColumnConstraints();
+        col3Constraints.setPercentWidth(47);
+        myDisplayArea.getColumnConstraints().addAll(col1Constraints, col2Constraints, col3Constraints);
+        myDisplayArea.setVgap(5);
+        myDisplayArea.setPadding(new Insets(0, 10, 10, 10));
+        HBox topheader = createHeader(header);
+        GridPane.setHgrow(topheader, Priority.ALWAYS);
+        thePane.add(topheader, 0, 0);
+        thePane.add(myDisplayArea, 0, 1);
+
+        return thePane;
+    }
+
+    public GridPane getDisplayArea()
+    {
+        return myDisplayArea;
+    }
+
+    public void setDisplayArea(GridPane myDisplayArea)
```

Comment:
Java docs, don't start method parameters with "my"

Reply:
changed to displayArea